UnitsNet.Tests/UnitSystemTests.cs

```diff
@@ -72,12 +72,14 @@ private enum CustomUnit
             IEnumerable<TUnitType> unitValues)
             where TUnitType : Enum
         {
+            var culture = GetCulture(cultureName);
```

Comment:
We should rename/split this test class too.

Reply:
👍 done!